src/main/java/com/amihaiemil/docker/UserAgentRequestHeader.java

```diff
@@ -56,11 +60,33 @@
                 String.join(
                     " ",
                     "docker-java-api /",
-                    VERSION,
+                    version(),
                     "See https://github.com/amihaiemil/docker-java-api"
                 )
             )
         ));
     }
 
+    /**
+     * Read current version from property file.
+     * @return Build version.
+     */
+    private static String version() {
+        final ClassLoader loader =
+            Thread.currentThread().getContextClassLoader();
+        final InputStream inputStream =
+            loader.getResourceAsStream(VERSION_FILE);
+        final String version;
+        final Properties properties = new Properties();
+        try {
+            properties.load(inputStream);
```

Comment:
Need to close `inputStream`, [specified stream remains open after this method returns](https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html#load(java.io.InputStream))

Reply:
@RobertoGraham Thanks, I'll fix this.